src/components/Autocomplete.vue

```diff
@@ -97,6 +98,12 @@ export default {
     inputClass: {
       type: [String, Object]
     },
+    /**
+     * CSS class for the input div itself
+     */
+    inputElementClass: {
```

Comment:
add documentation about this on README file

Reply:
done